### PR TITLE
1.18.1

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
Bump

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,30 +29,29 @@ requirements:
...
   requires:
     - pytest
-    - pytest-timeout
   commands:
     - f2py -h
-    - pytest --timeout=300 -v --pyargs numpy -k "not (test_loss_of_precision or test_einsum_sums_cfloat64)"  # [ppc64le]
-    - pytest --timeout=300 -v --pyargs numpy -k "not (test_loss_of_precision or test_may_share_memory_easy_fuzz or test_xerbla_override or test_may_share_memory_harder_fuzz or test_large_zip or test_sdot_bug_8577 or test_unary_ufunc_call_fuzz)"  # [aarch64]
-    - pytest --timeout=300 -v --pyargs numpy  # [not (aarch64 or ppc64le)]
+    - export OPENBLAS_NUM_THREADS=1  # [unix]
+    - set OPENBLAS_NUM_THREADS=1     # [win]
```
Comment (rgommers) [inline recipe/meta.yaml]: Why is this needed? I'd rather see the tests being run with the default settings.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,15 +1,12 @@
...
 
 package:
   name: numpy
   version: {{ version }}
 
 source:
   url: https://github.com/numpy/numpy/releases/download/v{{ version }}/numpy-{{ version }}.tar.gz
-  sha256: c93733dbebc2599d2747ceac4b18825a73767d289176ed8e02090325656d69aa
-  patches:
-    - macos-lto-check.patch
```
Comment (hmaarrfk) [inline recipe/meta.yaml]: Is this still needed?

@isuruf

**Comment by jakirkham** [comment]
Should this be closed now that we have `1.18.1` on `master`?
